server/controllers/user.js

```diff
@@ -21,3 +22,37 @@ exports.searchUsers = asyncHandler(async (req, res, next) => {
 
   res.status(200).json({ users: users });
 });
+
+const deletePhoto = async (user) => {
+  await cloudinary.uploader.destroy(user.profile_photo.public_id);
+  user.profile_photo = null;
+};
+
+exports.uploadProfilePhoto = asyncHandler(async (req, res, next) => {
+  const image = req.file;
+  const user = await User.findById(req.user.id);
+  if (!user) {
+    return res.sendStatus(403);
+  }
```

Comment:
403 really isn't the right status code for this kind of response. The user couldn't be found so a 404 is more appropriate and follows REST conventions more.